workflow_scheduler.go

```diff
@@ -1,18 +1,31 @@
 package main
 
 import (
+	"C"
 	"encoding/json"
 	"github.com/microsoft/BladeMonRT/workflows"
+	"github.com/microsoft/BladeMonRT/nodes"
 	"github.com/microsoft/BladeMonRT/logging"
 	"log"
-	winEvents "github.com/microsoft/BladeMonRT/windows_events"
+	win32 "github.com/0xrawsec/golang-win32/win32"
+	wevtapi "github.com/0xrawsec/golang-win32/win32/wevtapi"
+	"unsafe"
+	"github.com/microsoft/BladeMonRT/utils"
+	"github.com/google/uuid"
 )
 
 /** Class for scheduling workflows. */
 type WorkflowScheduler struct {
 	schedules []interface{}
 	logger *log.Logger
-	subscriber winEvents.EventSubscriber
+	eventSubscriptionHandles []wevtapi.EVT_HANDLE
+	guidToContext map[string]*CallbackContext
+}
+
+/** Class that represents a query for subscribing to a windows event. */
+type WinEventSubscribeQuery struct {
+	channel string
+	query string
 }
 
 /** Class for the schedule description in the JSON. */
```

Comment:
Rename "Workflow" field to "WorkflowName"

Reply:
Done